web-ui/src/main/resources/catalog/style/gn_icons.less

```diff
@@ -1,6 +1,6 @@
 /* this is a set of icons used in geonetwork, they either map to fa-variables or can be defined here (as font symbol or background image) */
-@import "../lib/style/font-awesome/less/variables";
-@import "../lib/style/font-awesome/less/mixins";
+@import "fa_variables";
```

Comment:
this file is missing @MichelGabriel no?

```

WARNING: Failed to compile less resource: file:/data/dev/gn/4/web-ui/src/main/resources/catalog/views/default/less/gn_admin_default.less.
Apr 19, 2023 7:20:31 PM org.fao.geonet.wro4j.GeonetLessCompilerProcessor process
WARNING: 1:2 The variable "@fa-css-prefix" was not declared..
```

Reply:
I somehow knew I was missing something 😬 . I've added it.